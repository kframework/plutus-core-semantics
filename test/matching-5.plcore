(program

(module Foo (imported) (exported () ()) (local () ())
  (define result
    (case (con Nat.Pair 1 2)
      (Nat.Pair (x y) (con Nat.Pair y x)))))

)
