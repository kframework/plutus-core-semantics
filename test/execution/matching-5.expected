<T> <k> . </k> <lenv> .Map </lenv> <genv> ( Foo . result ) |-> ( con Nat . Pair 2  ( 1  .ValList ) ) </genv> <mod> Foo </mod> </T>
